vavr/src/main/java/io/vavr/collection/TreeMap.java

```diff
@@ -1349,7 +1349,7 @@ public int size() {
 
     @Override
     public Seq<V> values() {
-        return iterator().map(Tuple2::_2).toStream();
+        return iterator().map(Tuple2::_2).toVector();
```

Comment:
This catches my eye - the only Map impl that overrides values(). Will double-check, if it is really necessary...